src/styles/theme/oc-drop.scss

```diff
@@ -0,0 +1,6 @@
+.oc-drop {
+  width: 250px;
+  border: 1px solid $muted-background;
+  background-color: $background;
+  box-sizing: border-box;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5880368)